nml/actions/action0.py

```diff
@@ -1119,11 +1119,9 @@ def get_callback_flags_actions(feature, id, flags):
     if not isinstance(prop_info_list, list):
         prop_info_list = [prop_info_list]
     for prop_info in prop_info_list:
-        act0.prop_list.append(
-            Action0Property(
-                prop_info["num"], parse_property_value(prop_info, expression.ConstantNumeric(flags)), prop_info["size"]
-            )
-        )
+        parsed_value = parse_property_value(prop_info, expression.ConstantNumeric(flags))
+        if "skip" not in prop_info or parsed_value[0].value != 0:
```

Comment:
I don't think the `skip` stuff is needed.
Not outputting flag prop when value is 0 should be fine with all features, and would be a small improvement in grf size.

Reply:
So you mean just never output a 0 cb prop, no matter if old or new prop?

Reply:
Yes, I think it should be fine as the prop value is or'ed or sb'ed on read (at least in openttd)